.gitignore

```diff
@@ -31,4 +31,3 @@ storage-layouts/proposed*
 target
 **/*.rs.bk
 test-ledger
-idls
```

Comment:
External IDLs are fetched locally on build, but we need them posted in the repo as `solana-verify verify-from-repo` would depend on them